src/components/ble/MusicService.cpp

```diff
@@ -56,63 +54,35 @@ namespace {
-  characteristicDefinition[2] = {.uuid = &msTrackCharUuid.u,
-                                 .access_cb = MusicCallback,
-                                 .arg = this,
-                                 .flags = BLE_GATT_CHR_F_WRITE | BLE_GATT_CHR_F_READ};
-  characteristicDefinition[3] = {.uuid = &msArtistCharUuid.u,
-                                 .access_cb = MusicCallback,
-                                 .arg = this,
-                                 .flags = BLE_GATT_CHR_F_WRITE | BLE_GATT_CHR_F_READ};
-  characteristicDefinition[4] = {.uuid = &msAlbumCharUuid.u,
-                                 .access_cb = MusicCallback,
-                                 .arg = this,
-                                 .flags = BLE_GATT_CHR_F_WRITE | BLE_GATT_CHR_F_READ};
-  characteristicDefinition[5] = {.uuid = &msPositionCharUuid.u,
-                                 .access_cb = MusicCallback,
-                                 .arg = this,
-                                 .flags = BLE_GATT_CHR_F_WRITE | BLE_GATT_CHR_F_READ};
-  characteristicDefinition[6] = {.uuid = &msTotalLengthCharUuid.u,
-                                 .access_cb = MusicCallback,
-                                 .arg = this,
-                                 .flags = BLE_GATT_CHR_F_WRITE | BLE_GATT_CHR_F_READ};
-  characteristicDefinition[7] = {.uuid = &msTotalLengthCharUuid.u,
-                                 .access_cb = MusicCallback,
-                                 .arg = this,
-                                 .flags = BLE_GATT_CHR_F_WRITE | BLE_GATT_CHR_F_READ};
-  characteristicDefinition[8] = {.uuid = &msTrackNumberCharUuid.u,
-                                 .access_cb = MusicCallback,
-                                 .arg = this,
-                                 .flags = BLE_GATT_CHR_F_WRITE | BLE_GATT_CHR_F_READ};
-  characteristicDefinition[9] = {.uuid = &msTrackTotalCharUuid.u,
-                                 .access_cb = MusicCallback,
-                                 .arg = this,
-                                 .flags = BLE_GATT_CHR_F_WRITE | BLE_GATT_CHR_F_READ};
-  characteristicDefinition[10] = {.uuid = &msPlaybackSpeedCharUuid.u,
-                                  .access_cb = MusicCallback,
-                                  .arg = this,
-                                  .flags = BLE_GATT_CHR_F_WRITE | BLE_GATT_CHR_F_READ};
-  characteristicDefinition[11] = {.uuid = &msRepeatCharUuid.u,
-                                  .access_cb = MusicCallback,
-                                  .arg = this,
-                                  .flags = BLE_GATT_CHR_F_WRITE | BLE_GATT_CHR_F_READ};
-  characteristicDefinition[12] = {.uuid = &msShuffleCharUuid.u,
-                                  .access_cb = MusicCallback,
-                                  .arg = this,
-                                  .flags = BLE_GATT_CHR_F_WRITE | BLE_GATT_CHR_F_READ};
+  characteristicDefinition[0] = {
```

Comment:
This is suspicious. What configuration are you using? What clang-format version?

Reply:
I'm just using clang-format version 13.0.1. I'm just using the clang-format used by this repository. The exact command-line arguments are `clang-format -i --style=file "$file"`.

Reply:
It might be differences between versions, I'll see if clang-format 12 (the version used by the clang-format GH Actions) formats it any differently.

Reply:
The GH Actions run has finished, and it doesn't recognise that as a mistake in formatting, so it isn't a difference between versions.

Reply:
This is controlled by AllowAllArgumentsOnNextLine. I never liked it. I think we should change it.

Reply:
Done, I think it looks better now.